apps/omg_eth/lib/omg_eth/ethereum_height_monitor.ex

```diff
@@ -0,0 +1,204 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Eth.EthereumHeightMonitor do
+  @moduledoc """
```

Comment:
does the following make sense?

I see you didn't implement terminate/2 callback. but what if you did and raised an alarm from there?

if it's restarted and is able to fetch new height it would clear the alarm anyway.

Reply:
The Ethereum client is returning ok/error tuple though so nothing would crash and terminate/2 would never get called?

Reply:
cool. case closed!